docs/src/native/simplenonlinearsolve.md

```diff
@@ -0,0 +1,59 @@
+# SimpleNonlinearSolve.jl
+
+These methods can be used independently of the rest of NonlinearSolve.jl
+
+```@index
+Pages = ["simplenonlinearsolve.md"]
+```
+
+## Interval Methods
+
+These methods are suited for interval (scalar) root-finding problems,
+i.e. `IntervalNonlinearProblem`.
+
+```@docs
+ITP
+Alefeld
+Bisection
+Falsi
+Ridder
+Brent
+```
+
+## General Methods
+
+These methods are suited for any general nonlinear root-finding problem, i.e.
+`NonlinearProblem`.
+
+| Solver                               | In-place | Out of Place | Non-Allocating (Scalars) | Non-Allocating (`SArray`) |
+|:------------------------------------ |:-------- |:------------ |:------------------------ |:------------------------- |
+| [`SimpleNewtonRaphson`](@ref)        | ✔️       | ✔️           | ✔️                       | ✔️                        |
+| [`SimpleBroyden`](@ref)              | ✔️       | ✔️           | ✔️                       | ✔️                        |
+| [`SimpleHalley`](@ref)               | ❌       | ✔️           | ✔️                       | ❌                         |
```

Comment:
ugh why is this one odd 😅

Reply:
I don't think so, the initial formatting came from my default markdown formatted in vscode.

Reply:
actually JuliaFormatter formats it back to the original one before your commit.